lib/reactotron-react-native/src/helpers/parseURL.test.ts

```diff
@@ -0,0 +1,59 @@
+import { getHostFromUrl } from "./parseURL"
+
+describe("getHostFromUrl", () => {
+  it("should throw when no host is found", () => {
+    expect(() => {
+      getHostFromUrl("")
+    }).toThrow()
+  })
+
+  it("should get host from URL without scheme", () => {
+    Object.entries({
+      localhost: "localhost",
+      "127.0.0.1": "127.0.0.1",
```

Comment:
do we need to match on ipv6?

Reply:
Lol, this was the 🐰 🕳️ I was trying to avoid 😜 I have an idea for tomorrow.

Reply:
Hey, fair enough, you can tell me you don't think we need it! I have no idea whether we do or not.

Reply:
I've added IPv6 and `exp://` schema for Expo Go.

Reply:
`exp://` may not be needed, but I included it as when you run `expo start` it says that metro is waiting on `exp://192.168.1.x:8081` so if that comes through on the `scriptURL` we'll handle it.

I added specs for both IPv6 and `exp` schema.